.github/workflows/build.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, macos-12, macos-latest, windows-latest]
+        os: [ubuntu-20.04, macos-12, macos-latest, windows-latest]
```

Comment:
Is there any need to support both 20.04 and latest?

Reply:
No. Any build on Ubuntu 20.04 will work on later versions but a build on the latest might not work on 20.04.